src/client/opamClient.ml

```diff
@@ -153,8 +154,9 @@ let details_of_package_regexps t packages ~exact_name ~case_sensitive regexps =
 
 let with_switch_backup command f =
```

Comment:
Does it make sense to have a backup only for the current switch when several can be modified (e.g. through `upgrade_all`) ?